challenges-webapp/src/main/webapp/vue-app/engagement-center/components/EngagementCenter.vue

```diff
@@ -44,6 +48,21 @@ export default {
   data: () => ({
     engagementCenterEnabled: eXo.env.portal.engagementCenterEnabled,
     tab: null,
+    isAdministrator: false,
   }),
+  created() {
+    this.isAdministratorCall();
+  },
+  methods: {
+    isAdministratorCall() {
+      fetch(`${eXo.env.portal.context}/${eXo.env.portal.rest}/gamification/realizations/api/allRealizations?fromDate=2022-07-31T23:00:00.000Z&toDate=2022-08-22T22:59:00.000Z&sortBy=date&sortDescending=true&offset=0&limit=11`)
```

Comment:
you should add this fetch call inside JS `realizationService.js` to be able to reuse it in other components when needed. you can call it after with `this.$realizationService...`

Reply:
service has been added successfully !